apps/api/src/server/migration/migrators/s51-advice-migrator.js

```diff
@@ -132,3 +141,20 @@ const mapStatus = (status) =>
 		depublished: 'not_checked',
 		notchecked: 'not_checked'
 	}[status?.replaceAll(' ', '')]);
+
+const generateAdviceTitleForFo = ({ enquiryMethod, enquirer, firstName, lastName }) => {
+	const adviceNameParams = { enquirer, firstName, lastName };
+	return enquiryMethod === 'meeting'
+		? `Meeting with ${generateAdviceName(adviceNameParams)}`
+		: `Advice to ${generateAdviceName(adviceNameParams)}`;
+};
+
+const generateAdviceName = ({ enquirer, firstName, lastName }) => {
+	if (enquirer) {
+		return enquirer;
+	} else if (firstName && lastName) {
```

Comment:
Do we want to return just firstName and lastName is isolation if only one of those exists?

Reply:
good spot, addressed!